xml/System.Speech.Recognition/SemanticResultKey.xml

```diff
@@ -315,7 +315,7 @@ private void keyTest()
       <Parameters />
       <Docs>
         <summary>Returns an instance of <see cref="T:System.Speech.Recognition.GrammarBuilder" /> constructed from the current <see cref="T:System.Speech.Recognition.SemanticResultKey" /> instance.</summary>
-        <returns>To be added.</returns>
+        <returns>Returns an instance of <see langword="GrammarBuilder"/> constructed from the current <see langword="SemanticResultKey"/> instance.</returns>
```

Comment:
This was actually missing on MSDN as well. but here's my suggestion for this one:

```suggestion
        <returns>An instance of <see cref="T:System.Speech.Recognition.GrammarBuilder" /> constructed from the current <see langword="SemanticResultKey"/> instance.</returns>
```